node-module-system/calculate.js

```diff
@@ -0,0 +1,32 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const inputs = [process.argv[3], Number(process.argv[2]), Number(process.argv[4])];
```

Comment:
Clever approach. I'm not convinced, though, that it makes the code easier to read than using separate variables.

Is `runTime(array)` easier to understand than `runTime(operation, arg1, arg2)`? And in the `runTime` code, is it easier to understand that `array[0]` is the operation or that the variable `operation` is the operation?